src/xcode/ENA/ENA/Source/Scenes/OptionGroup/DatePickerOption/__tests__/DatePickerOptionViewModelTests.swift

```diff
@@ -0,0 +1,104 @@
+		XCTAssertEqual(viewModel.datePickerDays[13], .past(Date(timeIntervalSinceReferenceDate: -8 * 24 * 60 * 60)))
+		XCTAssertEqual(viewModel.datePickerDays[14], .past(Date(timeIntervalSinceReferenceDate: -7 * 24 * 60 * 60)))
+		XCTAssertEqual(viewModel.datePickerDays[15], .past(Date(timeIntervalSinceReferenceDate: -6 * 24 * 60 * 60)))
+		XCTAssertEqual(viewModel.datePickerDays[16], .past(Date(timeIntervalSinceReferenceDate: -5 * 24 * 60 * 60)))
+		XCTAssertEqual(viewModel.datePickerDays[17], .past(Date(timeIntervalSinceReferenceDate: -4 * 24 * 60 * 60)))
+		XCTAssertEqual(viewModel.datePickerDays[18], .past(Date(timeIntervalSinceReferenceDate: -3 * 24 * 60 * 60)))
+		XCTAssertEqual(viewModel.datePickerDays[19], .past(Date(timeIntervalSinceReferenceDate: -2 * 24 * 60 * 60)))
+		XCTAssertEqual(viewModel.datePickerDays[20], .past(Date(timeIntervalSinceReferenceDate: -1 * 24 * 60 * 60)))
+		XCTAssertEqual(viewModel.datePickerDays[21], .today(Date(timeIntervalSinceReferenceDate: 0)))
+		XCTAssertEqual(viewModel.datePickerDays[22], .future(Date(timeIntervalSinceReferenceDate: +1 * 24 * 60 * 60)))
+		XCTAssertEqual(viewModel.datePickerDays[23], .future(Date(timeIntervalSinceReferenceDate: +2 * 24 * 60 * 60)))
+		XCTAssertEqual(viewModel.datePickerDays[24], .future(Date(timeIntervalSinceReferenceDate: +3 * 24 * 60 * 60)))
+		XCTAssertEqual(viewModel.datePickerDays[25], .future(Date(timeIntervalSinceReferenceDate: +4 * 24 * 60 * 60)))
+		XCTAssertEqual(viewModel.datePickerDays[26], .future(Date(timeIntervalSinceReferenceDate: +5 * 24 * 60 * 60)))
+		XCTAssertEqual(viewModel.datePickerDays[27], .future(Date(timeIntervalSinceReferenceDate: +6 * 24 * 60 * 60)))
+
+		XCTAssertEqual(viewModel.weekdays.count, 7)
+		XCTAssertEqual(viewModel.weekdays, ["M", "D", "M", "D", "F", "S", "S"])
+
+		XCTAssertEqual(viewModel.weekdayTextColors.count, 7)
+		XCTAssertEqual(viewModel.weekdayTextColors, [
+			.enaColor(for: .tint),
+			.enaColor(for: .textPrimary2),
+			.enaColor(for: .textPrimary2),
+			.enaColor(for: .textPrimary2),
+			.enaColor(for: .textPrimary2),
+			.enaColor(for: .textPrimary2),
+			.enaColor(for: .textPrimary2)
+		])
+    }
+
+	func testMonthChange() {
+		let viewModel = DatePickerOptionViewModel(
+			today: Date(timeIntervalSinceReferenceDate: 31 * 24 * 60 * 60)
+		)
+
+		XCTAssertEqual(viewModel.subtitle, "Januar–Februar 2001")
+	}
+
+	func testWithinOneMonth() {
+		let viewModel = DatePickerOptionViewModel(
+			today: Date(timeIntervalSinceReferenceDate: 28 * 24 * 60 * 60)
+		)
+
+		XCTAssertEqual(viewModel.subtitle, "Januar 2001")
+	}
+
+	func test() {
+		let viewModel = DatePickerOptionViewModel(
+			today: Date(timeIntervalSinceReferenceDate: 28 * 24 * 60 * 60),
+			calendar: .gregorian(with: .init(identifier: "en-US"))
+		)
+
+		XCTAssertEqual(viewModel.weekdays.count, 7)
+		XCTAssertEqual(viewModel.weekdays, ["S", "M", "D", "M", "D", "F", "S"])
```

Comment:
🤔 Shouldn't this return ["S", "M", "T", "W", "T", "F", "S"] with "en-US" ?

Reply:
We only change the region, not the language, that's why it's still the German localization with the American order.